healthcheck/tools/dcos/api/dcos.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2018 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"encoding/json"
+	"errors"
+	"time"
+
+	"github.com/percona/percona-server-mongodb-operator/healthcheck/pkg"
+	"github.com/valyala/fasthttp"
+)
+
+// SDKAPIVersion is the version of the DC/OS SDK API
+var SDKAPIVersion = "v1"
+
+// SDKClientName is the name of the DC/OS SDK API client
+var SDKClientName = "dcos"
+
+var (
+	DefaultHTTPTimeout   = "5s"
+	DefaultSchedulerHost = "api." + pkg.DefaultServiceName + ".marathon.l4lb.thisdcos.directory"
+	ErrEmptyBody         = errors.New("got empty body")
+	ErrNonSuccessCode    = errors.New("got non-success code")
+)
+
+// SDKClient is an HTTP client for the DC/OS SDK API
+type SDKClient struct {
+	config *Config
+	scheme HTTPScheme
+}
+
+// New creates a new SDKClient struct configured for use with the DC/OS SDK API
+func New(config *Config) *SDKClient {
+	c := &SDKClient{
+		config: config,
+		scheme: HTTPSchemePlain,
+	}
+	if config.Secure {
+		c.scheme = HTTPSchemeSecure
+	}
+	return c
+}
+
+func (c *SDKClient) Name() string {
+	return SDKClientName
+}
+
+func (c *SDKClient) get(url string, out interface{}) error {
+	req := fasthttp.AcquireRequest()
```

Comment:
why do you use fasthttp?
It can be done simpler with standard library
```
cl := &http.Client{
      Timeout: c.config.Timeout,
}
resp,err :=  cl.Post(url,"application/json",nil)
......
```